oasis/getting-started.md

```diff
@@ -96,6 +96,10 @@ used for multiple accounts.
    button below the label __ROOT PASSWORD__. Then click the __Open endpoint__
    button to bring up the ArangoDB web interface. Enter the credentials
    (user `root` and its password) and log in.
+   
+   {% hint 'info' %}
+   It is highly recommended to create additional users and not use the `root` user for everyday operations.
+   {% endhint %}
```

Comment:
Please add a link here to the https://deploy-preview-1008--zealous-morse-14392b.netlify.app/docs/3.9/programs-web-interface-users.html

Otherwise, lgtm!

Reply:
Will do that and update.